### PR TITLE
[MySQL] Add Dockerfile and instructions for Cloud Run

### PR DESCRIPTION
**Pull request by averikitsch**
Added new dependencies because of annoying logging warning.

### CODE REVIEW

File: cloud-sql/mysql/servlet/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use the official maven/Java 11 image to create a build artifact.
+# https://hub.docker.com/_/maven
+FROM maven:3.6.2-jdk-8-slim as builder
+
+# Copy local code to the container image.
+WORKDIR /app
+COPY pom.xml .
```
Comment (kurtisvg) [inline cloud-sql/mysql/servlet/Dockerfile]: Any particular reason for copying the POM separate?

File: cloud-sql/mysql/servlet/pom.xml [hunk 2]
```diff
@@ -64,14 +64,24 @@
       <artifactId>HikariCP</artifactId>
       <version>3.4.1</version>
     </dependency>
+    <dependency>
+       <groupId>org.slf4j</groupId>
```
Comment (kurtisvg) [inline cloud-sql/mysql/servlet/pom.xml]: Why are these added?

File: cloud-sql/mysql/servlet/Dockerfile (hunk 1 above)
Comment (averikitsch) [reply cloud-sql/mysql/servlet/Dockerfile]: We've been doing it across all samples in order to reduce the change of copying over files that aren't needed.

File: cloud-sql/mysql/servlet/pom.xml (hunk 2 above)
Comment (averikitsch) [reply cloud-sql/mysql/servlet/pom.xml]: There was an annoying logging warning about slf4j not being found. Though it doesn't break anything it was distracting. I can remove if needed.

Comment (kurtisvg) [reply cloud-sql/mysql/servlet/pom.xml]: LGTM, was just curious.
